mobify/pinny
# Updating pinny with appendTo option

Adding an `appendTo` option, which allows us to specify where to append the pinny.

Status: **Ready to merge**

Reviewers: @kpeatt @tedtate 
JIRA: https://mobify.atlassian.net/browse/CSOPS-921
Linked PRs: https://github.com/mobify/lockup/pull/6 - **THIS PR NEED MERGING FIRST**
## Changes
- adding `appendTo` option
- updating container reference to use lockup's instance property
## How to test-drive this PR
- checkout this branch
- `grunt test`
- `grunt serve` and make sure stuff isn't broken

TODO:
- [ ] Update lockup version number when branch is merged
